### PR TITLE
Support Django 1.11 beta

### LINKED ISSUE #70: RemovedInDjango20Warning: Using user.is_authenticated() and user.is_anonymous() as a method is deprecated.
permission/logics/author.py:103: RemovedInDjango20Warning: Using user.is_authenticated() and user.is_anonymous() as a method is deprecated.
Remove the parentheses to use it as an attribute.

Same for permission/logics/oneself.py:89

### LINKED ISSUE #71: Support Django 1.11

### PR DESCRIPTION
**Pull request by giginet**
closes #70 closes #71

### CODE REVIEW

**Comment by giginet** [comment]
@lambdalisue Please Review 🙏

File: src/permission/compat.py [hunk 1]
```diff
@@ -26,6 +26,19 @@ def isiterable(x):
 except ImportError:
     from django.utils.importlib import import_module
 
+if django.VERSION >= (1, 11):
```
Comment (lambdalisue) [inline src/permission/compat.py]: The `is_authenticated` and `is_anonymous` has become attributes from Django 1.10 (internally, it is a property which returns a callable boolean for backward compatibility [ref](https://github.com/django/django/blob/1.10/django/contrib/auth/base_user.py#L102))

So the line should be the following even if the method style is supported in Django 1.10

`if django.VERSION >= (1, 10):`

https://docs.djangoproject.com/en/1.10/ref/contrib/auth/#django.contrib.auth.models.User.is_authenticated

Comment (lambdalisue) [inline src/permission/compat.py]: The unusual condition should be in a IF block. I meant the code should be

```python
if django.VERSION < (1, 10):
    # For backward compatibility
    ...
else:
    # This is a default way
    ...
```

File: src/permission/compat.py [hunk 2]
```diff
@@ -26,6 +26,19 @@ def isiterable(x):
...
+    def is_authenticated(u):
+        return u.is_authenticated
```
Comment (lambdalisue) [inline src/permission/compat.py]: Please use `user` or `user_obj` instead of `u`.

**Comment by lambdalisue** [comment]
Thanks 👍

**Pull request merged**